### ISSUE TITLE
In webview, clicking links does nothing (wont navigate)

### ISSUE THREAD

**Issue by Overtorment** [react-native-community/react-native-webview#261]
Updated from 2.8.0 to latest, had to upgrade gradle along the way.
Getting this weird bug: 

![a href bug - peek 2019-01-19 03-33](https://user-images.githubusercontent.com/1913337/51421841-ca09d180-1b9c-11e9-9d30-cca55d4a0772.gif)

Everything worked on 2.8.0
Tried everything I could: bare minimum of props to webview; no wrap component for webview. Nothng helps.

Things that might be relevant: 

`Gradle 4.6`

` classpath 'com.android.tools.build:gradle:3.2.1'`

`"react-native": "^0.57.8",`

**Comment by johman10** [comment]
I'm experiencing the same problem on React Native Webview 5.2.1. After some debugging I found that assigning to the `window.location` doesn't do anything, which could be the source of this issue. I found this by following the [Debugging docs](https://github.com/react-native-community/react-native-webview/blob/master/docs/Debugging.md) and then running something like: `window.location = 'https://google.com'`. Just seems as if the Webview ignores this on Android for some reason.
